client/src/components/TaskReflectionModal/TaskReflectionModal.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import styles from "./TaskReflectionModal.module.css";
+import { PropTypes } from "prop-types";
+
+const RatingDot = ({ chosen, onChosen }) => {
```

Comment:
I would rename this to `selected` and `onSelected` respectively

Reply:
Renamed